src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs

```diff
@@ -734,7 +734,7 @@ private void CreateMempoolEntry(MempoolValidationContext context, long acceptTim
             // be mined yet.
             // Must keep pool.cs for this unless we change CheckSequenceLocks to take a
             // CoinsViewCache instead of create its own
-            if (!CheckSequenceLocks(this.network, this.chain.Tip, context, PowConsensusValidator.StandardLocktimeVerifyFlags, context.LockPoints))
+            if (!CheckSequenceLocks(this.network, this.chain.Tip, context, PowCoinviewRule.StandardLocktimeVerifyFlags, context.LockPoints))
```

Comment:
@dangershony, should the StandardLocktimeVerifyFlags be inside a rule? Maybe there is a better place for this? 
This variable is not used in the rule, only in block assembler and mempoolvalidator

Reply:
moving to MempoolValidator so that it ends up in a mempool rule eventually